<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firma Digital de Documentos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <!-- Bloque 1: Selección de PDF -->
    <item>
     <widget class="QWidget" name="widgetSeleccionPDF">
      <layout class="QHBoxLayout" name="horizontalLayoutPDF">
       <item>
        <widget class="QLabel" name="labelPDF">
         <property name="text">
          <string>Archivo PDF:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leRutaPdf">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExaminar">
         <property name="text">
          <string>Examinar...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <!-- Bloque 2: Previsualización -->
    <item>
     <widget class="QGraphicsView" name="gvPreview"/>
    </item>

    <!-- Bloque 3: Datos de firma -->
    <item>
     <widget class="QGroupBox" name="groupFirma">
      <property name="title">
       <string>Datos de firma</string>
      </property>
      <layout class="QFormLayout" name="formLayoutFirma">
       <item row="0" column="0">
        <widget class="QLabel" name="labelFirmante">
         <property name="text">
          <string>Nombre del firmante:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leFirmante"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelMotivo">
         <property name="text">
          <string>Motivo de la firma:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leMotivo"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="btnFirmar">
         <property name="text">
          <string>Aplicar firma al PDF</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

   </layout>
  </widget>

  <!-- Menú -->
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrirPdf"/>
    <addaction name="actionGuardarFirmado"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcercaDe"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>

  <widget class="QStatusBar" name="statusbar"/>

  <!-- Acciones -->
  <action name="actionAbrirPdf">
   <property name="text">
    <string>Abrir PDF</string>
   </property>
  </action>
  <action name="actionGuardarFirmado">
   <property name="text">
    <string>Guardar PDF firmado</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionAcercaDe">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>

 </widget>
 <resources/>
 <connections/>
</ui>
